### PR TITLE
Esnure the build target is called explicitly

### PR DESCRIPTION
**Pull request by mmitche**
The imported build tasks targets sets a nonexistent default target

### CODE REVIEW

**Comment by mmitche** [comment]
@jaredpar PTAL

File: src/Tools/MicroBuild/publish-blob.ps1 [hunk 1]
```diff
@@ -34,7 +34,7 @@ try {
         exit 1
     }
 
-    Exec-Console $msbuild "/p:ConfigDir=$configDir /p:ExpectedFeedUrl=$blobFeedUrl /p:AccountKey=$blobFeedKey /p:OutputPath=$configDir PublishBlobAssets.proj"
+    Exec-Console $msbuild "/p:ConfigDir=$configDir /p:ExpectedFeedUrl=$blobFeedUrl /p:AccountKey=$blobFeedKey /p:OutputPath=$configDir /t:Build PublishBlobAssets.proj"
```
Comment (jaredpar) [inline src/Tools/MicroBuild/publish-blob.ps1]: Why is this needed? That should be implicit ...

Comment (jasonmalinowski) [reply src/Tools/MicroBuild/publish-blob.ps1]: Indeed, @mmitche why is this required?

Comment (mmitche) [reply src/Tools/MicroBuild/publish-blob.ps1]: Because there is an error in the current build targets that reset the default target:

https://github.com/dotnet/buildtools/blob/280705d9d12af2fe0997509474825d2c9cbe7932/src/Microsoft.DotNet.Build.Tasks.Feed/PackageFiles/Microsoft.DotNet.Build.Tasks.Feed.targets#L1

Comment (mmitche) [reply src/Tools/MicroBuild/publish-blob.ps1]: This is just a workaround.  https://github.com/dotnet/buildtools/issues/1822
